cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -787,6 +787,19 @@ rules:
   - csidrivers
   verbs:
   - create
+- apiGroups:
```

Comment:
I think maybe this could be commented out with instructions to uncomment if users want to enable the admission webhook? What do you think about that @travisn ?

Reply:
If the admission controller is something that just runs as part of the operator, even if an optional component, I'd say to leave it uncommented here. We have this pattern already of enabling all the RBAC needed for the operator even if the operator doesn't have the features requiring the RBAC enabled.